install/kubernetes/cilium/values.yaml

```
@@ -1250,9 +1254,23 @@ sockops:
 
 # -- Configure TLS configuration in the agent.
 tls:
-  enabled: true
+  # -- This configures how the Cilium agent loads the secrets used TLS-aware CiliumNetworkPolicies
+  # (namely the secrets referenced by terminatingTLS and originatingTLS).
+  # Possible values:
+  #   - local
+  #   - tls
```

Oh, this is 100% on me, but this needs to be `k8s`. Sorry about that mistake, `tls` is not a valid value here.

```suggestion
  #   - k8s
```

---

all good :100:

---

I don't even know what this attribute is for before, thanks :)